### PR TITLE
feat: use token

### PR DESCRIPTION
**Pull request by aleacevedo**
## Summary

Implement useToken hook to use into the components

### CODE REVIEW

File: src/modules/controller/transaction.js [hunk 1]
```diff
@@ -321,6 +319,9 @@ const TransactionModule = (dispatch, getState) => {
       }
       const protectedIds = await getProtectedIds();
       const canisterInfo = app.whitelist[canisterId];
+
+      console.log('canisterInfo', JSON.stringify(canisterInfo, undefined, 2));
```
Comment (ignaciosantise) [inline src/modules/controller/transaction.js:323]: 👀

File: src/hooks/useToken.ts [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import { useSelector } from 'react-redux';
+
+import { getTokenPrices } from '@/constants/assets';
+import { State } from '@/interfaces/redux';
+import { getUsdAvailableAmount } from '@/screens/flows/Send/utils';
+
+export const useToken = (tokenId: string) => {
+  const { assets } = useSelector((state: State) => state.user);
+  const { icpPrice } = useSelector((state: State) => state.icp);
+  const token = assets.find(({ canisterId }) => canisterId === tokenId);
```
Comment (ignaciosantise) [inline src/hooks/useToken.ts:10]: is there a chance that the token is not found in the array? should we cover that case?

**Review by mattgle — APPROVED** [review]
Lgtm, test it and if everything ks working, merge it!

**Pull request merged**